yb-voyager/src/tgtdb/yugabytedb_test.go

```diff
@@ -123,15 +123,15 @@ func TestYugabyteGetPrimaryKeyColumns(t *testing.T) {
 		expectedPKCols []string
 	}{
 		{
-			table:          sqlname.NameTuple{CurrentName: sqlname.NewObjectName(POSTGRESQL, "test_schema", "test_schema", "foo")},
+			table:          testutils.CreateNameTupleWithTargetName("test_schema.foo", "public", POSTGRESQL),
 			expectedPKCols: []string{"id", "category"},
 		},
 		{
-			table:          sqlname.NameTuple{CurrentName: sqlname.NewObjectName(POSTGRESQL, "test_schema", "test_schema", "bar")},
+			table:          testutils.CreateNameTupleWithTargetName("test_schema.bar", "public", POSTGRESQL),
 			expectedPKCols: []string{"id"},
 		},
 		{
-			table:          sqlname.NameTuple{CurrentName: sqlname.NewObjectName(POSTGRESQL, "test_schema", "test_schema", "baz")},
+			table:          testutils.CreateNameTupleWithTargetName("test_schema.baz", "public", POSTGRESQL),
 			expectedPKCols: nil,
```

Comment:
case sensitivity and partition cases

Reply:
This is an old test. We can enhance it seperately

Reply:
adding these for GetIdentityColumnNamesForTables method